src/java.net.http/share/classes/java/net/http/HttpClient.java

```diff
@@ -410,6 +410,11 @@ public interface Builder {
         /**
          * Sets an authenticator to use for HTTP authentication.
          *
+         * @apiNote
+         * If a {@link HttpRequest} has an {@code Authorization} or {@code
+         * Proxy-Authorization} header set then its value will override any
+         * value derived from the given {@link Authenticator}.
+         *
```

Comment:
This sounds like we're going to call the authenticator, but we're not. Maybe:
```suggestion
         * @apiNote
         * If a {@link HttpRequest} has an {@code Authorization} or {@code
         * Proxy-Authorization} header set then its value will be used and
         * the {@link Authenticator} will not be invoked for the corresponding
         * authentication.
         *
```